tests/drupal-8/drupal_ti_test/composer.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "drupal/drupal_ti_test",
+    "type": "drupal-module",
+    "description": "Drupal Travis Integration - Test module.",
+    "homepage": "https://github.com/LionsAd/drupal_ti",
+    "license": "GPL-2.0+",
+    "repositories": [
+        {
+            "type": "composer",
+            "url":  "https://packages.drupal.org/8"
+        }
+    ],
+    "require": {
+      "drupal/core": "~8",
```

Comment:
D'oh. Well, I guess I'm glad that I left that mistake..show it actually _attempted_ to work.